packages/cli/src/metrics/prometheus-metrics.service.ts

```diff
@@ -141,7 +150,12 @@ export class PrometheusMetricsService {
 				'/form-waiting/',
 				'/form-test/',
 			],
-			metricsMiddleware,
+			(req, res, next) => {
```

Comment:
I think most of these are [configurable](https://github.com/n8n-io/n8n/blob/master/packages/@n8n/config/src/configs/endpoints.config.ts#L93), so I think we should use the paths coming from configs